apps/condo/tsconfig.json

```diff
@@ -27,6 +27,7 @@
   "exclude": [
     "node_modules",
     ".next",
-    "out"
+    "out",
+    "cypress/cypress.config.ts"
```

Comment:
you are actually included this, but PR title is about to exclude

Reply:
Fixed that